airbyte-webapp/src/components/LabeledSwitch/LabeledSwitch.module.scss

```diff
@@ -0,0 +1,30 @@
+@use "../../scss/colors";
+
+.labeledSwitch {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.label {
+  padding-left: 7px;
+  font-size: 13px;
+  line-height: 16px;
+  cursor: pointer;
+  color: colors.$textColor;
```

Comment:
@edmundito `textColor` feels like one of those named variables we might want to keep around? What are your thoughts around using this vs `darkBlue` here?

Reply:
And forget the question for this PR, since I (aka the CI) noticed, that's actually not a SCSS variable anymore :D So will be replaced here.